tests/test_jquery_installed.py

```diff
@@ -32,7 +32,7 @@ def inner(**kwargs):
 @pytest.mark.skipif(sphinx.version_info[:2] < (6, 0),
                     reason="Requires Sphinx 6.0 or greater")
 def test_jquery_installed_sphinx_ge_60(blank_app):
-    out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"]})
+    out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"], "jquery_cors_enable": True})
```

Comment:
Setting `jquery_cors_enable` here doesn't seem to work.

I'm suspecting this could be the same as https://github.com/sphinx-doc/sphinx/issues/10120 ?

Reply:
I've now tested on a local documentation build that this setting actually has an effect :)